package.json

```
@@ -3,10 +3,8 @@
   "version": "3.0.0-dev",
   "description": "Apache Cordova create module. Creates new project from default or template",
   "main": "index.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/apache/cordova-create.git"
-  },
+  "repository": "github:apache/cordova-create",
```

```suggestion
  "repository": "https://github.com/apache/cordova-create",
```

I would prefer a real URL so you can copy paste it to the browser. But short notation is fine, too.